plugins/base/src/main/java/edu/wpi/first/shuffleboard/plugin/base/widget/GraphWidget.java

```diff
@@ -117,29 +111,35 @@
   /**
    * How often graphs should be redrawn, in milliseconds.
    */
-  private static final long UPDATE_PERIOD = 250;
+  private static final long UPDATE_PERIOD = 200;
 
   static {
     ThreadUtils.newDaemonScheduledExecutorService()
         .scheduleAtFixedRate(() -> {
           synchronized (graphWidgets) {
+            // This method actually updates the graph (as seen by the user).
             graphWidgets.forEach(GraphWidget::update);
           }
         }, 500, UPDATE_PERIOD, TimeUnit.MILLISECONDS);
```

Comment:
This can be changed. The default NT update rate is 10Hz - can we match that? Can we get to 30Hz?

If it can be made faster, look into making it a setting on the base plugin so users can get faster or slower refresh times